charts/transaction-tool/README.md

```diff
@@ -0,0 +1,7 @@
+If you want to create a simple postgres single pod cluster for dev purposes simply set `stackgres.enabled` to `false`
```

Comment:
I believe this should support 3 modes:

- No DB
- Small Dev DB
- "Production" DB